__test__/file-transformer.test.js

```diff
@@ -0,0 +1,127 @@
+        expect(data.slice(1078, 11078).toString('hex')).toEqual(testAssets.invertedPixelArray.toString('hex'));
+        done();
+      });
+    });
+  });
+
+  describe('fileTransformer greyscale', () => {
+    it('the greyscale methods color table buffer values should be an average for each rgb set', done => {
+      fileTransformer(testAssets.buffer, ['greyscale'], function(err, data) {
+        expect(data.slice(54, 1078).toString('hex')).toEqual(testAssets.greyscaleColorTableBuffer.toString('hex'));
+        done();
+      });
+    });
+  });
+
+  describe('fileTransformer verticalmirror', () => {
+    it('the verticalmirror methods pixel array values should be reversed', done => {
+      fileTransformer(testAssets.buffer2, ['verticalmirror'], function(err, data) {
+        expect(data.slice(1078, 11078).toString('hex')).toEqual(testAssets.verticalmirrorPixelArray.toString('hex'));
+        done();
+      });
+    });
+  });
+
+  describe('fileTransformer redscale', () => {
+    it('the redscale methods color table buffer values of g in the rgb should be 0', done => {
+      fileTransformer(testAssets.buffer2, ['redscale'], function(err, data) {
+        expect(data.slice(54, 1078).toString('hex')).toEqual(testAssets.redscaleColorTableBuffer.toString('hex'));
+        done();
+      });
+    });
+  });
+
+  describe('fileTransformer bluescale', () => {
+    it('the bluescale methods color table buffer values of b in the rgb should be 0', done => {
+      fileTransformer(testAssets.buffer3, ['bluescale'], function(err, data) {
+        expect(data.slice(54, 1078).toString('hex')).toEqual(testAssets.bluescaleColorTableBuffer.toString('hex'));
+        done();
+      });
+    });
+  });
+
+  describe('fileTransformer greenscale', () => {
+    it('the greenscale methods color table buffer values of r in the rgb should be 0', done => {
+      fileTransformer(testAssets.buffer4, ['greenscale'], function(err, data) {
+        expect(data.slice(54, 1078).toString('hex')).toEqual(testAssets.greenscaleColorTableBuffer.toString('hex'));
+        done();
+      });
+    });
+  });
+
+  describe('fileTransformer whitereplace', () => {
+    it('the whitereplace methods color table buffer values of r in the rgb should be 0', done => {
+      fileTransformer(testAssets.buffer5, ['whitereplace'], function(err, data) {
+        expect(data.slice(54, 1078).toString('hex')).toEqual(testAssets.whitereplaceColorTableBuffer.toString('hex'));
+        done();
+      });
+    });
+  });
+
+  describe('fileTransformer blackreplace', () => {
+    it('the blackreplace methods color table buffer values of r in the rgb should be 0', done => {
+      fileTransformer(testAssets.buffer6, ['blackreplace'], function(err, data) {
+        expect(data.slice(54, 1078).toString('hex')).toEqual(testAssets.blackreplaceColorTableBuffer.toString('hex'));
+        done();
+      });
+    });
+  });
+
+  describe('fileTransformer invertedverticalmirror', () => {
```

Comment:
invertedverticalmirror and invertedhorizontalmirror don't match what is written in the README documentation. (horizontalmirrorinverted and verticalmirrorinverted) As a result, if user tries to run transforms using only the README documentation, two of the transforms seems unable to complete.